### PR TITLE
Add k8s build script

### PR DESCRIPTION
**Pull request by afshin-paydar**

### CODE REVIEW

File: k8s-build_tools/release.sh [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+# ( Could probably do this in one command, but I think it'd require eval'ing the output and we don't want to ever do that!
+DESIRED_REPLICAS=$(kubectl get hpa/${APP_NAME} -o jsonpath='{.status.desiredReplicas}')
+
+# But Desired replicas from the HPA can be < minimum, so we need to take that into account
+# Min Replicas over Current, in case there's an issue with the current deployment
+MIN_REPLICAS=$(kubectl get hpa/${APP_NAME} -o jsonpath='{.spec.minReplicas}')
+
+TARGET_REPLICAS=$(( DESIRED_REPLICAS > MIN_REPLICAS ? DESIRED_REPLICAS : MIN_REPLICAS ))
+
+# TODO - Setup the kubectl context so that kubectl works without arguments. We should not have to pass things like --server to it
```
Comment (ghost) [inline k8s-build_tools/release.sh]: @afshinpaydar-binary Think this needs to move to the top as a note of what we need to do before running this.

* Set namespace in context
* Setup target kube server
* Auth credentials

All of that should be ideally tokenised in context, or worse case scenario in env vars ( But I don't think we resort to that ).

HTH

**Review by ghost — APPROVED** [review]
@afshinpaydar-binary Great 👍 

One niggle to move the TODO up to the top and turn it into an expectation statement and examples

File: k8s-build_tools/release.sh (hunk 1 above)
Comment (afshin-paydar) [reply k8s-build_tools/release.sh]: Need to figure out how to do that!

Comment (afshin-paydar) [reply k8s-build_tools/release.sh]: We can simply rely on `.kube/config` if we don’t want to pass parameters to this script, I guess

**Review by 4p00rv — APPROVED** [review]

**Pull request merged**